test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# Script to validate the packaged module and configs
+
+# Ensure our workdir is that of the project root
+cd "${0%/*}/../.." || exit 
+
+TEST_NAME="Deployment testing"
+declare -a CONFIG_ARRAY
+if [[ $# -gt 0 ]]; then
+    for CONFIG in "$@"; do
+      CONFIG=$(ls {"$CONFIG",config/"$CONFIG",config/"$CONFIG".json,"$CONFIG".json} 2> /dev/null)
+      if [ ! -f "$CONFIG" ]; then
+        echo "Config file $CONFIG does not exist, check the path."
+        exit 1
+      fi
+      CONFIG_ARRAY+=("$CONFIG")
+    done
+    echo "Deployment testing ${#CONFIG_ARRAY[@]} configs:"
+else
+  mapfile -t CONFIG_ARRAY < <(ls config/*.json)
+  echo "Deployment testing all configs:"
+fi
+printf " - %s\n" "${CONFIG_ARRAY[@]}"
+
+mkdir -p .tmp
+echo "$TEST_NAME - Building and installing package"
+npm pack --loglevel warn
+npm install -g solid-community-server-*.tgz --loglevel warn
+rm solid-community-server-*.tgz
+
+run_server_with_config () {
+  if [[ $# -ne 2 ]]; then
+    echo "Config arguments not set"
+    return 1
+  elif [ ! -f "$1" ]; then
+    echo "Config file does not exist, check the path."
+    return 1
+  fi
+  CONFIG_PATH=$1
+  CONFIG_NAME=$2
+
+  mkdir -p .data
+
+  CSS_ARGS="-p 8888 -l warn -f .data/ -s http://localhost:4000/sparql"
+  CSS_BASE_URL="http://localhost:8888"
+
+  # HTTPS config specifics: self-signed key/cert + CSS base URL override
+  if [[ $CONFIG_NAME =~ "https" ]]; then
+    openssl req -x509 -nodes -days 1 -newkey rsa:2048 -keyout .tmp/server.key -out .tmp/server.cert -subj '/CN=localhost' &>/dev/null
```

Comment:
Didn't this already happen in the `ci.yml`?

Reply:
Yes but I'm going to remove it there. If we only do it in ci.yml we can't test:deploy locally. Unless we keep self-signed certs in our repo for testing, which is also an option.

Reply:
Removing from the CI is fine, was just confused why I saw it twice.